tedana/selection/select_comps.py

```diff
@@ -310,19 +357,20 @@ def selcomps(seldict, comptable, mmix, manacc, n_echos):
             candart)
         ign_add0 = np.setdiff1d(ign_add0, midk)
         comptable.loc[ign_add0, 'classification'] = 'ignored'
-        comptable.loc[ign_add0, 'rationale'] += 'I010;'
+        comptable.loc[ign_add0, 'rationale'] += 'I011;'
         ign = np.union1d(ign, ign_add0)
 
         ign_add1 = np.intersect1d(
             acc[comptable.loc[acc, 'kappa'] <= kappa_elbow],
             acc[comptable.loc[acc, 'variance explained'] > new_varex_lower])
         ign_add1 = np.setdiff1d(ign_add1, midk)
         comptable.loc[ign_add1, 'classification'] = 'ignored'
-        comptable.loc[ign_add1, 'rationale'] += 'I011;'
-        ign = np.union1d(ign, ign_add1)
-        acc = np.setdiff1d(acc, np.union1d(midk, ign))
+        comptable.loc[ign_add1, 'rationale'] += 'I012;'
+        ign = np.union1d(ign, ign_add1)  # unnecessary
```

Comment:
Should we just remove these two lines if they're unnecessary ?

Reply:
Yup, definitely. Will do.